draft-ietf-rats-corim.md

```diff
@@ -768,68 +769,79 @@ The types defined for a group identified are UUID and variable-length opaque byt
 {::include cddl/group-id-type-choice.cddl}
 ~~~
 
-##### Measurements
+##### Properties
+
+A Property is a key-value pair that in CoRIM represents a single piece of actual state or a possible state.
+
+A Property in CoRIM always appears within a triple.
+The triple expresses the intended meaning of the Property.
+In all cases, a Property is made by an authority, such as an Attester, Endorser, or Reference Value Provider.
+
+A Property from an Attester is called Evidence.
+A Property in a `reference-triple-record` is called a Reference Value.
+A Property in another triple is called an Endorsement.
+A Reference Value triple is expressible as a conditional Endorsement that if the Attester made some Properties, then the RVP also makes those Properties.
 
-Measurements can be of a variety of things including software, firmware,
-configuration files, read-only memory, fuses, IO ring configuration, partial
-reconfiguration regions, etc. Measurements comprise raw values, digests, or
-status information.
+A Property may be called an assertion, property, measurement, or many other classifications based on its role in Evidence appraisal.
```

Comment:
I assume that property and claim have to switch places here.

```suggestion
A Property may be called an assertion, claim, measurement, or many other classifications based on its role in Evidence appraisal.
```
And then, why do we say that here in the first place? Should that go to the definition of Property?